README.md

```diff
@@ -1,56 +1,181 @@
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>html</li>
+  </ul>
+</details>
+<details>
+  <summary>Style</summary>
+  <ul>
+    <li>Css</li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>none</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>None</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Zero html-css linters error]**
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Not available at the moment]()
+
+## 💻 Getting Started
+
+> Describe how a new developer could make use of your project.
 
 To get a local copy up and running, follow these steps.
+Clone this repository to your desired folder:
+
+````sh
+  cd Hello-Project
+  git https://github.com/ghermaico135/Hello-Project
+
+### Prerequisites
+
+In order to run this project you need:
+
+install a vscode text editor or any other editor
+chrome
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this commands:
+
+```sh
+  cd Hello-Project
+  git https://github.com/ghermaico135/Hello-Project
+````
+
+### Install
+
+Install this project with:
+
+Run this command:
 
-Prerequisites
-In order to run this project you need: to have latest chrome browser installed
+```sh
+  cd hello-microversehttps://github.com/ghermaico135/Hello-Project
+  node.js v12 and above
+```
 
-Setup
-Clone this repository to your desired folder:"github/ghermaico135"
-And try to open in any editor however recommended is vscode
+### Usage
 
-Usage
-To run the project, execute right click open in chrome browser
+To run the project, execute the following command:
 
-#Run tests
-I used html-css linter to run the test for minimize errors and to improve performance
-#Author
-Author miki
-GitHub: ghermaico135
-Twitter: @mikidarling
-LinkedIn: miki
-🔭 Future Features
-new feature-1 will be added to make more dynamic by linking with javascript
+Example command:
+
+```sh
+  use git bash to open in Vscode
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+Example command:
+
+```sh
+  Run "npx stylelint "**/*.{css,scss}" --fix " to fix linters
+```
+
+### Deployment
+
+You can deploy this project using:
+
+Github Page
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/ghermaico135)
+- Twitter: [@twitterhandle](https://twitter.com/@mikidarling)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/miki)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
```

Comment:
- [ ] Please consider removing the default placeholders eg: `Describe 1 - 3 features you will add to the project`. This will improve your readme and make it more professional.